cibuildwheel/ios.py

```diff
@@ -206,14 +209,33 @@ def cross_virtualenv(
     #
     # To prevent problems, set the PATH to isolate the build environment from
     # sources that could introduce incompatible binaries.
+    #
+    # However, there may be some tools on the path that are needed for the
+    # build. Find their location on the path, and link the underlying binaries
+    # (fully resolving symlinks) to a "safe" location that will *only* contain
+    # those tools. This avoids needing to add *all* of Homebrew to the path just
+    # to get access to (for example) cmake for build purposes.
+    safe_tools_path = venv_path / "cibw_safe_tools"
+    safe_tools_path.mkdir()
+    for tool in safe_tools:
+        tool_path = shutil.which(tool)
+        if tool_path is None:
+            msg = f"Could not find a {tool!r} executable on the path."
+            raise errors.FatalError(msg)
```

Comment:
(if we do set a default, this would be:)
```suggestion
            continue
```

Reply:
I'm seeing this a bit differently.
If it's been requested in the configuration, then fail, it's a default tool then continue. (this would probably mean the default list does not initialize safe-tools but rather extends it).